index.html

```diff
@@ -87,7 +87,19 @@
       href: "https://tools.ietf.org/html/draft-grigorik-http-client-hints",
       status: "Internet-Draft",
       publisher: "IETF",
-    }
+    },
+    "CLIENT-HINTS-INFRASTRUCTURE": {
```

Comment:
We should probably move this over to https://github.com/WICG/admin/blob/gh-pages/biblio.json

Reply:
Will do in a followup!